### ISSUE TITLE
Dataset Details View: Generate citation suggestion automatically

### ISSUE THREAD

**Issue by NoYo25** [BEXIS2/Core#827]
Feature request from Explos.

- [x] check, if this a wish also from other instances (any overlap or totally different)(@jitendragaikwad )
- [x] check if all required system keys would exist (@aostrow )
  - Author (ok)
  - Version (ok)
  - Title (ok)
  - Year (generated from what?)
  - DOI (**need to be added!**)
  -  publisher (from settings)
  - Id (ok)
  - ???
- [x] check if there is some overlap with the pending schema.org issue (same keys?)
- [x] decide on output format (Text, BibTeX, ...?) -> BibTex (conversion UI side)
- [x] deal with first & last name (https://github.com/BEXIS2/Core/issues/894)
- [x] basic implementation backend (https://github.com/BEXIS2/Core/issues/893)
- [x] #2139

**Is your feature request related to a problem? Please describe.**
* The problem I am facing is the dataset citation. I have followed **How To Cite a Dataset** guidelines where it asks for collecting the data creators with a ";" separator. I faced some datasets with more than 20 names. Creating a citation by manually copying and pasting these names from the dataset metadata view is a very time-consuming task. I have almost done with 6 datasets in 2 hours or more.

**Describe the solution you'd like**
* In the metadata view of datasets, it would be beneficial to add "cite" btn.
It supposes to generate latex entry for the dataset following the normal rules of citing a dataset. 
In other words, such a button automates the **How To Cite a Dataset** guidelines by generating a latex entry for users.

**Additional context**
* This is might be an example of the expected output of  **cite** button:
```
@dataset{bexis_25126,
	author       = {
    	Markus Fischer and
        Thomas Nauss  and		
    	Marco Tschapka and
        Wolfgang Weisser and
        Jörg Müller	
	},
	title        = {{Aggregated species richness and habitat heterogeneity variables for testing the habitat-heterogeneity hypothesis, 2006-2018 }},	
	year         = 2020,
	publisher    = {Biodiversity Exploratories Information System},
	version      = 2,
	doi          = {10.25829/bexis.25126-1},	
}
```

**Comment by geofranzi** [comment]
@EleonoraPetzold  Das kommt glaube dem Plan das mal in den Core zu integreren am nächsten. Daher hab ich dich da mal hinzugefügt.

**Comment by DavidBlaa** [comment]
is implementet
